docs/labels.md

```diff
@@ -0,0 +1,22 @@
+# Project-specific labels
+
+## Component
+
+We have a set of compiled binaries which also enclose a subset of functionality
+(there are overlaps for internal components, but those are OK), so they
+naturally form this set of labels:
+
+- neofs-adm
+- neofs-cli
+- neofs-ir
+- neofs-lens
+- neofs-storage
```

Comment:
"storage" intentionally?

Reply:
Yeah, it's named that way and I'm not changing it.